src/Line.cpp

```diff
@@ -7,64 +7,64 @@
 
 namespace rtt {
 
-double Line::distanceToLine(const Vector2 &point) const {
-    return (this->project(point) - point).length();
-}
+    double Line::distanceToLine(const Vector2 &point) const {
+        return (this->project(point) - point).length();
+    }
 
 ///Computes the projection of point onto the line. This is identical to picking the closest point on the line
```

Comment:
Should probably move this to the header.

Reply:
I don't see why, could you elaborate? It's just a member implementation, why not keep it in the translation unit

Reply:
I meant the comment on line 14, not the implementation.